spec/features/applications/remission_confirmation_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+RSpec.feature 'Confirmation page for remission', type: :feature do
+
+  include Warden::Test::Helpers
+  Warden.test_mode!
+
+  let(:user) { create :user, office: create(:office) }
+  let(:application) { create(:application_part_remission) }
+
+  def visit_confirmation_page
+    visit application_build_path(application_id: application.id, id: 'confirmation')
+  end
+
+  context 'as a signed in user', js: true do
+    before do
+      WebMock.disable_net_connect!(allow: ['127.0.0.1', 'codeclimate.com', 'www.google.com/jsapi'])
+      Capybara.current_driver = :webkit
+      json = '{"original_client_ref": "unique", "benefit_checker_status": "No",
+                  "confirmation_ref": "T1426267181940",
+                  "@xmlns": "https://lsc.gov.uk/benefitchecker/service/1.0/API_1.0_Check"}'
+      stub_request(:post, "#{ENV['DWP_API_PROXY']}/api/benefit_checks").with(body:
+      {
+        birth_date: application.date_of_birth.strftime('%Y%m%d'),
+        entitlement_check_date: application.date_received.strftime('%Y%m%d'),
+        id: "#{user.name.gsub(' ', '').downcase.truncate(27)}@#{application.created_at.strftime('%y%m%d%H%M%S')}.#{application.id}",
+        ni_number: application.ni_number,
+        surname: application.last_name.upcase
+      }).to_return(status: 200, body: json, headers: {})
+      login_as user
```

Comment:
There's a lot of repetition of this sort between all the feature specs. We should refactor this to a helper and automatically include it. 

But it's not relevant directly to this story.